Installation/cmake/modules/UseLASLIB.cmake

```diff
@@ -1,4 +1,4 @@
 # This module setups the compiler for using LASlib library.
 # It assumes that find_package(LASLIB) was already called.
 
-add_definitions(-DCGAL_LINKED_WITH_LASLIB)
+add_definitions(-DCGAL_LINKED_WITH_LASLIB -DUSE_AS_DLL)
```

Comment:
It's from _LASlib_.

We forked it, so I could change it, but I'd rather change as little things as possible in the library. Maybe I can add `#define USE_AS_DLL` inside the CGAL code just before including the LASlib header and `#undef` right after (there is exactly one inclusion of LASlib). What do you think?